pypga/core/interface/remote/server.py

```diff
@@ -30,8 +30,9 @@ def __init__(self, host, port=2222, delay=0.05, bitstreamfile=None, start=True):
             delay=delay,
         )
         self.stop()
-        self.run(f"mkdir {self._destpath}")
-        self.run(f"cd {self._destpath}")
+        destpath=str(self._destpath).replace("\\","/")
```

Comment:
I suppose the cleaner solution is simply to make `destpath` of type `pathlib.PosixPath`, so it uses posix conventions rather than the one of the system running the code.

Reply:
Yes. I was convinced that something like this exists.